spacy_transformers/layers/trfs2arrays.py

```diff
@@ -21,33 +22,65 @@ def forward(model: Model, trf_datas: List[TransformerData], is_train: bool):
     grad_factor = model.attrs["grad_factor"]
     outputs = []
     backprops = []
+
+    # If possible, cache the width for allocating zero-length tensors.
```

Comment:
Hmmm, I am not a big fan of this, since it is only conditionally right. I think it's nicer to be predictable and be right or wrong all the time.

Luckily, this won't be an issue in curated transformers.

Reply:
Hmm, fair point. I'll leave the width at zero in all cases then.